node/hyperbahn/service_proxy.js

```diff
@@ -124,6 +124,9 @@ ServiceDispatchHandler.prototype.rateLimit =
 function rateLimit(req, buildRes) {
     var self = this;
 
+    // stats all traffic using rate limiter's counter
+    self.rateLimiter.incrementServiceCounter(req.headers.cn + '~~' + req.serviceName);
```

Comment:
We also shouldnt rate limit edges. 

This is a lot of coupling and nuance here.